machine_learning_experiments/assets/categorized-data.yml

```diff
@@ -948,3 +1027,31 @@
   topics: []
 - url: "https://modrinth.com/mod/trufflez-arcanum"
   topics: []
+- url: "https://openpyxl.readthedocs.io/en/latest/print_settings.html"
+  topics: []
+- url: "https://openpyxl.readthedocs.io/en/latest/datetime.html"
+  topics: []
+- url: "https://streamlit-book.readthedocs.io/en/latest/"
+  topics: []
+- url: "https://doorstop.readthedocs.io/en/latest/"
+  topics: []
+- url: "https://doorstop.readthedocs.io/en/latest/cli/interchange/"
+  topics: []
+- url: "https://origins.readthedocs.io/en/latest/types/damage_condition_types/attacker/"
+  topics: []  # Game dev
+- url: "https://androidaps.readthedocs.io/en/latest/Usage/Enabling-Engineering-Mode-in-xDrip.html"
+  topics: []  # Maybe mobile
+- url: "https://hikari-lightbulb.readthedocs.io/en/latest/index.html"
+  topics: []
+- url: "https://hikari-lightbulb.readthedocs.io/en/latest/guides/application-commands.html"
+  topics: []
+- url: "https://google-calendar-simple-api.readthedocs.io/en/latest/events.html"
```

Comment:
I'm a little hesitant to tag pages like this as no topic, since they have a ton of Python code on them. It definitely feels like this would be very confusing to me if you told me this wasn't backend, but then told me other pages with random Python code were.

Reply:
I think this requires more discussion. Certainly the argument could be made that most Python stuff is "backend" although it isn't exactly web.

Reply:
Yea.. I think it's probably just a symptom of having non-exclusive topics. There's always going to be something that matches 80% nothing and 20% an existing topic, or similar. 🤷 

I think it's fine to move ahead here, since this is a specific instance of a larger problem, and we'll just continue to refine to get closer to a solution over time.